xcode/Subconscious/Shared/Components/Deck/CardStackView.swift

```diff
@@ -287,8 +283,9 @@ struct CardStack: View {
             Spacer()
             GeometryReader { geo in
                 ZStack {
-                    ForEach(enumeratedDeck, id: \.element.id) {
-                        index, card in
+                    ForEach(deck.indices, id: \.self) { index in
+                        let card = deck[index]
+                        
```

Comment:
Apparently there are problems with using `enumerated()` this way that can result in `BAD_ACCESS` errors, see: https://stackoverflow.com/a/63145650

Using `indices` is apparently safer and seems equally readable.